### PR TITLE
Issue #774 by JrgenGNielsen: Add creator and materialtype to reservat…

### PR DESCRIPTION
**Pull request by JrgenGNielsen**
…ion- & loan list

Issue #774 by JrgenGNielsen: Add .

Issue #774 by JrgenGNielsen: Add compiled stylesheets

Issue #774 by JrgenGNielsen: Add compiled and compressed stylesheets

Issue #774 by JrgenGNielsen: Update stylesheets

### CODE REVIEW

File: modules/ding_base/templates/material_item.tpl.php [hunk 1]
```diff
@@ -8,9 +8,16 @@
...
+      <h4 class="item-material-type"><?php print $material_type; ?></h4>
+    <?php endif; ?>
   </div>
   <div class="right-column">
-    <h3 id="<?php print $availability_id; ?>" class="item-title"><?php print $title; ?></h3>
+    <?php if (!empty($creators)) : ?>
+      <h4 class="item-creators"><?php print $creators; ?></h4>
+    <?php endif; ?>
+    <h3 id="<?php print $availability_id; ?>"
+        class="item-title"><?php print $title; ?></h3>
```
Comment (cableman) [inline modules/ding_base/templates/material_item.tpl.php]: This seam to be a wired line-back in relation to the rest of the code's style?

**Comment by cableman** [comment]
Only minor HTML style issue.

Everything else looks good :-).

File: modules/ding_base/templates/material_item.tpl.php (hunk 1 above)
Comment (JrgenGNielsen) [inline modules/ding_base/templates/material_item.tpl.php]: @cableman : I'm not sure I understand what a 'line-back' is?

Comment (guddo) [inline modules/ding_base/templates/material_item.tpl.php]:
```
<h3 id="<?php print $availability_id; ?>"
        class="item-title"><?php print $title; ?></h3>
```

vs

`<h3 id="<?php print $availability_id; ?>" class="item-title"><?php print $title; ?></h3>`

**Comment by cableman** [comment]
Sorry for the typo... should have been line-break.

**Comment by cableman** [comment]
Merged by hand to resolve conflict in compiled css.

**Pull request merged**